debian/rules

```diff
@@ -21,3 +21,11 @@ override_dh_auto_install:
 	mkdir -p debian/nfd/etc/avahi/services/
 	cp debian/avahi.service debian/nfd/etc/avahi/services/nfd.service
 
+override_dh_installinit:
+	dh_systemd_enable -pnfd --name=nfd nfd.service
+	dh_systemd_enable -pnfd --name=nfd-init nfd-init.service
+	dh_installinit -pnfd --no-start --noscripts
+	dh_systemd_start -pnfd --no-restart-on-upgrade
```

Comment:
BTW, are you sure you don't want to restart? If there's a bug in NFD, wouldn't you want it to get restarted?